api_docs/actions.mdx

```diff
@@ -12,7 +12,7 @@ import actionsObj from './actions.json';
 
 
 
-
+Contact [Kibana Alerting](https://github.com/orgs/elastic/teams/kibana-alerting-services) for questions regarding this plugin.
```

Comment:
FYI, these URLs 404 for people outside the @elastic org. Maybe we should consider rewording it asking people to ping `@kibana-alerting-services`.